base_layer/core/src/transactions/transaction_components/unblinded_output_builder.rs

```diff
@@ -63,6 +68,19 @@ impl UnblindedOutputBuilder {
         }
     }
 
+    pub fn update_recovery_byte_if_required(
```

Comment:
this is at the builder stage, so no need to be cautious

```suggestion
    pub fn update_recovery_byte(
```